### PR TITLE
Pull Request for adding style in Bookstore

### PR DESCRIPTION
**Pull request by chuaindia**
In this branch,

-  styling is done for the bookstore. 

- ZEPLIN is used as a tool.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,14 +1,281 @@
...
   display: flex;
   justify-content: space-around;
+  width: 55rem;
+  height: 10.625rem;
+  margin: 2.313rem 9.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
 }
```
Comment (Wes-Isaac) [inline src/App.css]: - [x] Great job so far, kindly check the alignment of the design on Zeplin and try to resemble that.

### Design
Screenshot 2022-11-16 at 2 58 47 PM

### Your implementation
Screenshot 2022-11-16 at 3 02 25 PM

File: src/App.css [hunk 2]
```diff
@@ -1,14 +1,281 @@
...
+
+.LogoBorder {
+  width: 2.813rem;
+  height: 2.813rem;
+  border-radius: 50%;
+  position: absolute;
+  left: 90%;
+  background: none;
+  border: solid 1px #e8e8e8;
+}
```
Comment (Wes-Isaac) [inline src/App.css]: - [x] Great job, kindly center the icon.

### Design
image

### Your Implementation
image

File: src/App.css [hunk 3]
```diff
@@ -1,14 +1,281 @@
...
+  border: none;
+  padding: 0;
+  font-size: 0.8rem;
+  color: #fff;
+  background-color: #0290ff;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.BookForm {
```
Comment (Wes-Isaac) [inline src/App.css:301]: - [x] Great job, kindly make sure the spacing for the form inputs resemble the design, also check the font size and font weight.
### Design
image


### Your Implementation
Screenshot 2022-11-16 at 3 06 43 PM

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @chuaindia, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional README 👍.
- Great Implementation 💯.
- Used correct workflow 🥇.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/App.css (hunk 1 above)
Comment (chuaindia) [reply src/App.css]: Ok.

File: src/App.css (hunk 2 above)
Comment (chuaindia) [reply src/App.css]: Ok.

File: src/App.css (hunk 1 above)
Comment (chuaindia) [reply src/App.css:18]: Necessary changes made compared with Zeplin as :

  width: **75**rem;
  height: 10.625rem;
  margin: 2.313rem **6.25rem** 0;
  padding: 2rem 9.188rem 1.625rem 1.688rem;
  border-radius: 4px;
  border: solid 1px #e8e8e8;
  background-color: #fff;

For the right side the following implementation is done following Zeplin:

.Books-Detail {
  width: 100rem;
  height: 89.625rem;
  padding: 9.313rem 3.75rem 7.688rem 6.25rem;
  background-color: var(--pale-grey);
}

File: src/App.css (hunk 2 above)
Comment (chuaindia) [reply src/App.css:42]: Changes made following Zeplin style, according to the Reviewer's review :

.LogoBorder {
  width: 2.813rem;
  height: 2.813rem;
  margin: 0 0 0 41.5rem;
  padding: 0.875rem;
  border: solid 1px #e8e8e8;
}

File: src/App.css (hunk 3 above)
Comment (chuaindia) [reply src/App.css:301]: Space is made accordingly.

File: src/App.css (hunk 1 above)
Comment (chuaindia) [reply src/App.css]: Correction made accordingly.

File: src/App.css (hunk 2 above)
Comment (chuaindia) [reply src/App.css]: Correction made accordingly.

**Review by uwadonat — APPROVED** [review]
## Hi @chuaindia,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by chuaindia** [comment]
A lot of thanks to you.

**Pull request merged**